src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -332,23 +332,26 @@ def update_container_branches() {
     withEnv([]){
         sh """
         set +x
+	shopt -s extglob
         source /home/saluser/.setup_dev.sh || echo loading env failed. Continuing...
 
         for repo in \$(ls /home/saluser/repos/)
         do
             cd /home/saluser/repos/\$repo
             /home/saluser/.checkout_repo.sh ${WORK_BRANCHES}
-            git pull
```

Comment:
Why this loop? I see the exact same code in the next loop as well. Can you just delete this one? If not, then please add a code comment explaining what's going on (to help future maintainers).